sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/main/java/com/azure/messaging/eventhubs/checkpointstore/blob/BlobCheckpointStore.java

```diff
@@ -197,12 +249,6 @@ private PartitionOwnership convertToPartitionOwnership(BlobItem blobItem) {
                 case OWNER_ID:
                     partitionOwnership.setOwnerId(value);
                     break;
-                case SEQUENCE_NUMBER:
-                    partitionOwnership.setSequenceNumber(Long.valueOf(value));
-                    break;
-                case OFFSET:
-                    partitionOwnership.setOffset(Long.valueOf(value));
-                    break;
                 default:
                     // do nothing, other metadata that we don't use
                     break;
```

Comment:
Same as the other comment about using metadata as a map instead of iterating over the entries.